server/views/index.pug.orig

```diff
@@ -0,0 +1,352 @@
+doctype html
```

Comment:
Duplicate file?

Reply:
Yes, apologies, all these .orig files - I thought they will be removed automatically at some point after I've finished the merge. I've now removed them